### DISCUSSION TITLE
Using image.Thumbnail compared to image.Resize

### DISCUSSION THREAD

**Discussion by SmartieCodes** [dlemstra/Magick.NET#816] [Help] [answered]
What are the differences between using .thumbnail and .resize?

Does .thumbnail attempt to use the EXIF thumbnail if it exists and is large enough?

Thanks

**Comment by dlemstra** [comment, marked as answer]
You can find the answer to your first question here: https://imagemagick.org/Usage/resize/. The thumbail from the exif data will not be used. You could use `ThumbnailLength` and `ThumbnailOffset` to create a thumbnail from the `ExifProfile` class yourself. I could probably add a method for that in the future.
